feature-libs/checkout/components/components/payment-type/payment-type.component.ts

```diff
@@ -30,7 +30,7 @@ export class PaymentTypeComponent {
   typeSelected$: Observable<string> = this.paymentTypeService
     .getSelectedPaymentType()
     .pipe(
-      filter((selected) => selected !== undefined),
+      filter((selected: any) => selected !== undefined),
```

Comment:
Perhaps instead of `any` we use the return type of `getSelectedPaymentType()` that is `string | undefined`